ta/gcov/include/ta_gcov.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 NXP
+ */
+
+#ifndef TA_GCOV_H
+#define TA_GCOV_H
+
+#define TA_GCOV_UUID { 0xa424901c, 0x4810, 0x4e4d, \
+		{ 0xaa, 0x76, 0x62, 0xef, 0x00, 0x45, 0x34, 0x3f} }
+
+#define TA_GCOV_CMD_GET_VERSION	0
+#define TA_GCOV_CMD_DUMP_CORE	1
+#define TA_GCOV_CMD_DUMP_TA	2
```

Comment:
Can you provide the ABI

Here, it would be:
```c
/*
 * TA_GCOV_CMD_GET_VERSION - Proxy function which calls gcov_get_version()
 *
 * [out]    value[0].a	    version of gcov
*/
#define TA_GCOV_CMD_GET_VERSION	0

/*
 * TA_GCOV_CMD_DUMP_CORE - Proxy function which calls gcov_get_version()
 *
 * [out]    value[0].a	    version of gcov
*/
#define TA_GCOV_CMD_DUMP_CORE	1

 *          [in]     memref[0]	    description


 *          [in]     memref[0]	    filepath
 *          [in]     memref[1]	    code coverage data
#define TA_GCOV_CMD_DUMP_TA	2

Reply:
Sure